private-config.js

```diff
@@ -0,0 +1,16 @@
+let config = {
+  rinkeby: {
+    keystore: process.env.RINKEBY_KEYSTORE
```

Comment:
Are these environment variables intended to be used elsewhere, perhaps in test run scripts or something like that? Or are these environment variables set automatically by the Truffle framework? I am curious how these are defined for use elsewhere.